docs/network_firewall.rst

```diff
@@ -322,38 +365,133 @@ to configure the OPT1 interface. Go to **Interfaces ▸ OPT1**, and check
+The easiest way to set up your firewall rules is to
+look at the screenshots of a correctly configured firewall and
 edit the interfaces, aliases, and firewall rules on your firewall to
 match them.
 
+Use Screenshots of Firewall Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here are some example screenshots of a working pfSense firewall
+configuration. You will add the firewall rules until they match what is
+shown on the screenshots.
+
+First, we will configure IP and port aliases. Navigate to **Firewall ▸ Aliases**
+and you should see a screen with no currently defined IP aliases:
+
+|Blank IP Aliases|
+
+Next you will click **Add** to add each IP alias.
+You should leave the **Type** as **Host**.
+Make aliases for the following:
+
+- ``admin_workstation``: ``10.20.1.2``
+- ``app_server``: ``10.20.2.2``
+- ``external_dns_servers``: ``8.8.8.8, 8.8.4.4``
+- ``local_servers``: ``app_server, monitor_server``
+- ``monitor_server``: ``10.20.3.2``
+
+|Add Firewall Alias|
+
+Click **Save** to add the alias.
+
+Keep adding aliases until the screenshot matches what is shown here:
+
+|Firewall IP Aliases Pre Save|
+
+Finally, hit **Apply Changes**. This will save your changes. You should see a
+message "The changes have been applied successfully":
+
+|Firewall IP Aliases Post Save|
+
+Next click "Ports" for the port aliases, and add the following ports:
+
+- OSSEC: ``1514``
+- ossec_agent_auth: ``1515``
+
+Your configuration should match this screenshot:
+
+|Port Aliases|
+
+Next we will configure firewall rules for each interface. Navigate to **Firewall ▸
+Rules** to add firewall rules for the LAN, OPT1, and OPT2 interfaces.
+
+.. warning:: Be sure not to delete the Anti-Lockout Rule on the LAN interface.
+    Deleting this rule will lock you out of the pfSense WebGUI.
+
+Add or remove rules until they match the following screenshots by clicking **Add**
+to add a rule.
+
+**LAN interface:**
+
+|Firewall LAN Rules|
+
+**OPT1 interface:**
+
+|Firewall OPT1 Rules|
+
+**OPT2 interface:**
+
+|Firewall OPT2 Rules|
+
+Once you've set up the firewall, exit the Unsafe Browser, and continue
+with the "Keeping pfSense up to date" section below.
+
+SecureDrop Configuration (Templates)
+------------------------------------
+
+Alternatively, you can load the provided ``.xml`` templates.
+
+First navigate to **Diagnostics ▸ Backup & Restore**:
+
+|Backup & Restore|
+
+Scroll down to "Restore Backup" and install each one of the template files for
+**Restore Areas** Interfaces, Aliases, and Firewall Rules, in that order:
+
+- Interfaces config: ``install_files/network_firewall/interfaces-config-pfSense.xml``
+- Aliases: ``install_files/network_firewall/aliases-config-pfSense.xml``
+- Firewall rules: ``install_files/network_firewall/filter-config-pfSense.xml``
+
+Note that none of the template filters are encrypted. Click "Restore Configuration"
+to restore each file. After this is done, verify that the rules have taken
```

Comment:
I understand the expression "have taken", but maybe "have been restored properly" would be clearer here.